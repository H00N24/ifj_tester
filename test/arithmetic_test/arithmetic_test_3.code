// Pretypovanie pri operacich s int a double

class Main
{
	static void run()
	{
		double a = 2.457878;
		double c = a * 2;

		double i = (c - 5) * 20;

		ifj16.print("a: " + a + "\n");
		ifj16.print("c: " + c + "\n");
		ifj16.print("Celkovo: " + i + "\n");
	}
}
