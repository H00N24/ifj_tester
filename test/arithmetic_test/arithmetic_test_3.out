a: 2.45788
c: 4.91576
Celkovo: -1.68488
